src/ResourceManager/Insights/Commands.Insights/LogProfiles/RemoveAzureRmLogProfileCommand.cs

```diff
@@ -39,8 +37,18 @@ public class RemoveAzureRmLogProfileCommand : ManagementCmdletBase
 
         protected override void ProcessRecordInternal()
         {
-            WriteWarning("The output of this cmdlet will change. The cmdlet will not return anything in future releases.");
-            this.InsightsManagementClient.LogProfiles.DeleteAsync(logProfileName: this.Name, cancellationToken: CancellationToken.None).Wait();
+            WriteWarning("The type of the output will change to follow the pattern from other similar operations.");
+            Rest.Azure.AzureOperationResponse result = this.MonitorManagementClient.LogProfiles.DeleteWithHttpMessagesAsync(logProfileName: this.Name, cancellationToken: CancellationToken.None).Result;
+
+            /*
+             * This object will be returned in future releases
+            var response = new AzureOperationResponse
+            {
+                RequestId = result.RequestId,
+                StatusCode = result.Response != null ? result.Response.StatusCode : HttpStatusCode.OK
+            };
+			*/
+
             WriteObject(true);
```

Comment:
@gucalder this cmdlet should implement [PassThru](https://github.com/Azure/azure-powershell/wiki/PowerShell-Cmdlet-Design-Guidelines#outputtype)

[Here is an example of a cmdlet using `PassThru`](https://github.com/Azure/azure-powershell/blob/preview/src/ResourceManager/DataLakeAnalytics/Commands.DataLakeAnalytics/Commands/RemoveAzureRmDataLakeAnalyticsAccount.cs#L39)